mathlibtools/lib.py

```diff
@@ -149,7 +149,7 @@ def get_mathlib_archive(rev: str, url:str = '', force: bool = False,
     """
 
     # we check for xz archives first
-    fnames = [rev + '.tar.xz', rev + '.tar.gz', rev + '.tar.bz2']
+    fnames = [rev + '.tar.xz']
```

Comment:
There is no point in keeping a list with only one element, this function must be simplified more drastically.

Reply:
I pushed a commit that addressed this.